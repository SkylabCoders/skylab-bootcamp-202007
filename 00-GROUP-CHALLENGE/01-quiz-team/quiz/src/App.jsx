import React from 'react';
import './css/app.scss';
import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
import PageNotFound from './components/PageNotFound';
import Header from './components/Header';
import Footer from './components/Footer';
import LeaderBoard from './components/LeaderBoard';
import LoginScreen from './components/LoginScreen';
import Dashboard from './components/Dashboard';
import Question from './components/Question';
import Game from './components/Game';
import Answer from './components/Answer';
import * as ROUTES from './config/routes';

function App() {
	return (
		<div className="app">
			<Router>
			<Header />
			<Switch>
				<Route path={ROUTES.HOME} exact component={Dashboard} />
				<Route path={ROUTES.LEADERBOARD} component={LeaderBoard} />
				<Route path={ROUTES.LOGIN} component={LoginScreen} />
				<Route path={ROUTES.QUESTION} component={Question} />
				<Route path={ROUTES.THEMES_WILDCARD} component={Game} />
				<Route path={ROUTES.ANSWER} component={Answer} />
				<Route component={PageNotFound} />
			</Switch>
			<Footer />
			</Router>
		</div>
	);
}

export default App;
