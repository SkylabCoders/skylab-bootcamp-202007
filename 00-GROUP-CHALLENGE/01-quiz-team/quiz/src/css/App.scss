.app {
  width: 100%;
}

/*# sourceMappingURL=App.scss.map */
