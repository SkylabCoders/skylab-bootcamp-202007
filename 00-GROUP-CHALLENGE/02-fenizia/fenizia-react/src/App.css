* {
  margin: 0;
  padding: 0;
  box-sizing: border-box;
  font-family: 'Lato', sans-serif;
}


.margin-body {
  padding-top: 60px;
  min-height: calc(100vh - 140px - 0px); 
}
