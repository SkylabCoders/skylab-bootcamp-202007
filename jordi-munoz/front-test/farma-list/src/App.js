import React from 'react';
import { Route, Switch, BrowserRouter } from 'react-router-dom';
import ProductList from './components/ProductList';

function App() {
  return (
    <div className="App">
      <BrowserRouter>
        <Switch>
          <Route path="/" exact component={ProductList} />
        </Switch>
      </BrowserRouter>
    </div>
  );
}

export default App;
