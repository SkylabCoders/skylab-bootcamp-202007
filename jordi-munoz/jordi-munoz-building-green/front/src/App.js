import React from 'react';
import { Route, Switch, BrowserRouter } from 'react-router-dom';
import Home from './components/Home';
import CategorySlider from './components/CategorySlider';
import Header from './components/Header';
import Results from './components/Results';
import Footer from './components/Footer';
import RegisterProject from './components/RegisterProject';
import Login from './components/Login';
import Profile from './components/Profile';
import RegisterUser from './components/RegisterUser';



function App() {
  return (
    <div className="App">
      <BrowserRouter>
        <Header />
        <Switch>
          <Route path='/' exact component={Home} />
          <Route path='/form' component={CategorySlider} />
          <Route path='/results' component={Results} />
          <Route path='/project-register' component={RegisterProject} />
          <Route path='/login' component={Login} />
          <Route path='/user-register' component={RegisterUser} />
          <Route path='/profile' component={Profile} />
        </Switch>
        <Footer />
      </BrowserRouter>
    </div>
  );
}

export default App;


