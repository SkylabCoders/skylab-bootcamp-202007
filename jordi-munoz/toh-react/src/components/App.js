import React from 'react';
import HeroDetail from './HeroDetail';
import HeroDashboard from './HeroDashboard';
import HeroList from './HeroList';
import Header from './Header';
import { Route, Switch, Redirect } from 'react-router-dom';
import PageNotFound from './PageNotFound';


function App(props) {
	return (
		<div className="container">
			<Header />
			<Switch>
				<Route path="/" exact component={HeroDashboard} />
				<Route path="/hero/:heroId" component={HeroDetail} />
				<Route path="/hero" component={HeroList} />
				<Redirect from="/heroes" to="/hero" />
				<Route component={PageNotFound} />
			</Switch>
		</div>
	);
}

export default App;
