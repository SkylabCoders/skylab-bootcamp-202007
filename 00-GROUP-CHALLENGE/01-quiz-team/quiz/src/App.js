import React from 'react';
import './css/App.css';
import { Route, Switch } from 'react-router-dom';
import PageNotFound from './components/PageNotFound'
import Home from './components/Home'
import Header from './components/Header'
import Footer from './components/Footer'
import LeaderBoard from './components/LeaderBoard'
import LoginScreen from './components/LoginScreen'
import Dashboard from './components/Dashboard'


function App() {
	return (
		<div className="app">
			<Header />
			<Switch>
				<Route path="/" exact component={Dashboard} />
				<Route path="/leaderboard" exact component={LeaderBoard} />
				<Route path="/login" exact component={LoginScreen} />
				<Route component={PageNotFound} />
			</Switch>
			<Footer />
		</div>
	);
}

export default App;
