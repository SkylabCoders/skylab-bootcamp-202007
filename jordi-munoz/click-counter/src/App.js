import React from 'react';
import './App.css';

function App({ clicks, clickChange, mouseLeave }) {
  return <div 
  onClick={clickChange} 
  onMouseLeave={mouseLeave}
  >
  This div has been clicked {clicks} times!
  </div>;

}

export default App;
