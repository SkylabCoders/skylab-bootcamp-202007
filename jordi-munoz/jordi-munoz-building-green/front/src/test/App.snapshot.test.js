import renderer from 'react-test-renderer';
import React from 'react';
import App from '../App';


describe('App snapshot', () => {
  const tree = renderer.create(<App />);

  it('should match', () => {
    expect(tree.toJSON()).toMatchSnapshot();
  });

});