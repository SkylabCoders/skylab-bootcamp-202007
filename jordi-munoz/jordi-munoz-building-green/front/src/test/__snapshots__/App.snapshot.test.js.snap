// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot should match 1`] = `
<div
  className="App"
>
  <div
    className="header-container"
  >
    <div
      className="header"
    >
      <a
        aria-current="page"
        className="logo active"
        href="/"
        onClick={[Function]}
        style={Object {}}
      >
        <img
          alt="Logo"
          className="header-container__logo"
          src="logo-blanco.png"
        />
      </a>
      <input
        className="menu-btn"
        id="menu-btn"
        type="checkbox"
      />
      <label
        className="menu-icon"
        htmlFor="menu-btn"
      >
        <span
          className="navicon"
        />
      </label>
      <ul
        className="menu"
      >
        <li>
          <a
            href="/"
          >
            HOME
          </a>
        </li>
        <li>
          <a
            href="/project-register"
          >
            Proyectos
          </a>
        </li>
        <li>
          <a
            href="/results"
          >
            Resultados
          </a>
        </li>
        <li>
           
          <button
            className="loginButton"
            onClick={[Function]}
          >
            Log In
          </button>
        </li>
      </ul>
    </div>
  </div>
  <div
    style={
      Object {
        "marginBottom": "60px",
        "marginTop": "80px",
      }
    }
  >
    <div
      className="home-container"
    >
      <img
        alt="Logo"
        className="home-container__logo"
        src="logo-bg.png"
      />
      <p>
        Cada vez más las personas estamos más concienciados de la importancia de la sostenibilidad. Esta importancia se transmite en la elección de la ciudad y la vivienda donde queremos vivir.
        <br />
        <br />
        Los manuales de sostenibilidad son una herramienta para evaluar la sostenibilidad, la calidad de vida y la eficiencia energética de las viviendas. 
        <br />
        <br />
        Desde Espai Energy, realizamos un manual de sostenibilidad acorde con las necesitades del cliente y cogiendo referencias de las principales certificaciones sostenibles globales, como LEED, BREEAM, WELL o PASSIVHAUS.
      </p>
      <img
        alt="Certificaciones"
        src="certifications.png"
        width="400px"
      />
      <p>
        Desde esta aplicación el usuario puede evaluar su edificio/promoción en el manual de sostenibilidad de su empresa.
        <br />
        <br />
        El manual está compuesto, tal como se puede ver en la siguiente imagen, por diversos créditos sostenibles clasificados en varias categorías de sostenibilidad.
        <br />
        <br />
        En función del cumplimiento de cada criterio se obtendrán puntos, al final de toda la evaluación se sumarán los puntos y se obtendrá una clasificación sostenible (≥ 50 puntos edificio sostenible, ≥ 75 puntos edificio verde).
      </p>
      <img
        alt="Categorias"
        src="categoriesList.png"
      />
    </div>
  </div>
  <div
    className="container"
  >
    <p>
      made by Jordi Muñoz for:  
    </p>
    <a
      href="https://.espaienergy.com/"
    >
      <img
        alt="Espai Energy Logo"
        src="logo_ee_H.png"
        title="Espai Energy SL"
      />
    </a>
  </div>
</div>
`;
